### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by rex4539**

### CODE REVIEW

**Comment by rex4539** [comment]
@pwnall Could you please review?

File: CMakeLists.txt [hunk 1]
```diff
@@ -364,7 +364,7 @@ if(CRC32C_BUILD_TESTS)
     set_property(TARGET crc32c_capi_tests APPEND PROPERTY COMPILE_OPTIONS "/WX")
 
     # The Windows SDK version currently on CI produces warnings when some
-    # headers are #included using C99 compatibity mode or above. This workaround
+    # headers are #included using C99 compatibility mode or above. This workaround
```
Comment (pwnall) [inline CMakeLists.txt:367]: I think this line is over 80-characters now. Please wrap around to the next line?

**Review by pwnall — CHANGES_REQUESTED** [review]
Thank you for spotting the typo!

File: CMakeLists.txt (hunk 1 above)
Comment (rex4539) [reply CMakeLists.txt]: Fixed.

**Review by pwnall — APPROVED** [review]
LGTM, thank you for the revisions!

**Pull request merged**